diesel_codegen/Cargo.toml

```diff
@@ -10,7 +10,7 @@ repository = "https://github.com/diesel-rs/diesel/tree/master/diesel_codegen"
 keywords = ["orm", "database", "postgres", "sql", "codegen"]
 
 [dependencies]
-diesel_codegen_syntex = { version = "0.7.0", default-features = false }
+diesel_codegen_syntex = { path = "../diesel_codegen_syntex", version = "0.7.3", default-features = false }
```

Comment:
Dropping support for Syntex versions is definitely a breaking change. This would need to be in 0.8.0. However, pull requests shouldn't bump the version we'll do it when it's time to release.